### PR TITLE
Fix punctuation

### PR DESCRIPTION
**Pull request by glenacota**
Changes in this pull request mainly enforce consistency in lists punctuation - i.e., end sentences with a period. Other changes fix punctuation typos.

### CODE REVIEW

**Review by DJRickyB — COMMENTED** [review]
@glenacota Thank you so much for the PR! Left one ask so far

File: docs/track.rst [hunk 1]
```diff
@@ -485,7 +485,7 @@ If we do not want to reuse these operations, we can also define them inline. Not
       ]
     }
 
-Contrary to the ``query``, the ``force-merge`` operation does not take any parameters, so Rally allows us to just specify the ``operation-type`` for this operation. It's name will be the same as the operation's type::
+Contrary to the ``query``, the ``force-merge`` operation does not take any parameters, so Rally allows us to just specify the ``operation-type`` for this operation. Its name will be the same as the operation's type::
```
[the remark this answers is not in the document]
Comment (glenacota) [reply docs/track.rst:488]: There are many double columns, I suspect is part of a tag to declare an example.

[the remark this answers is not in the document]
Comment (DJRickyB) [reply docs/track.rst:488]: my mistake, yes please leave this

File: docs/track.rst [hunk 2]
```diff
@@ -19,7 +19,7 @@ A track is specified in a JSON file.
 Ad-hoc use
 ..........
 
-For ad-hoc use you can store a track definition anywhere on the file system and reference it with ``--track-path``, e.g::
+For ad-hoc use you can store a track definition anywhere on the file system and reference it with ``--track-path``, e.g.:
```
Comment (DJRickyB) [inline docs/track.rst:22]: this double-colon should be preserved https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#literal-blocks

**Review by DJRickyB — APPROVED** [review]
Looks great!  Very timely as we are just about to push a new release (including for our docs), much appreciated

**Comment by glenacota** [comment]
thanks for your amazing job!

**Pull request merged**